specs/Print.md

```diff
@@ -0,0 +1,425 @@
+
+Print API
+===
+
+# Background
+
+Developers have requested the ability to programmatically print the current page using a print dialog, print without a print dialog in WebView2. Currently end user can open a print preview dialog by pressing "Ctrl+ P" or selecting "Print" from context menu in a document and to open system print dialog requires pressing 
+"Ctrl + Shift + P" or clicking "Print using system dialog" link in print preview dialog. This API gives developers the ability to open print dialog without requiring the end user interaction and also can print without a dialog to a specific printer by configuring the print settings. 
+
```

Comment:
Would be good to mention the existing CoreWebView2.PrintToPdf and why its not good enough (prints to a file which isn't allowed for some customers like banks, and no print dialog which is hard to remedy in a cross-platform manner on their own)